### DISCUSSION TITLE
bad-effects-of-sugar/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#54] [Announcements]
# bad-effects-of-sugar/

همون‌طور که احتمالاً می‌دونید اخیراً در حال مطالعه در مورد سلامتی هستم و جالب این که هر چی بیشتر مطالعه می‌کنم، بیشتر متوجه میشم که قند تقریباً «هیچ» خاصیتی نداره و منشأ خیلی از بیماٰری‌های ما می‌تونه همین قند بیش از حدی باشه که مصرف می‌کنیم.
یعنی جز حالت‌های خاص که مثلاً کسی قند خونش پایین اومده که در اون صورت سرم حاوی قند بهش تزریق می‌کنند، فایده دیگه‌ای نمی‌تونم براش متصور بشم.

https://aprd.ir/bad-effects-of-sugar/

**Comment by masoudnasrollahi** [comment]
عسل و خرما و انجیر هم تاثیر مشابهی دارند؟!

**Reply by pourmand1376** [reply]
سلام مسعود جان. 

اینطور که من متوجه شدم، تو طبیعت قند معمولاً به همراه فیبر وجود داره که باعث میشه اون اثرات بدی که گفتم رو نداشته باشه یا خیلی خیلی کمتر باشه، در حدی که دیگه انقدر مهم نیست؛ یعنی بدن ما تکامل یافته که هر چیزی که می‌خوره به همراهش پادزهرش (فیبر) رو هم بخوره که براش مشکلی پیش نیاد. و خب وقتی فیبر می‌خوریم خیلی وقت‌ها دیگه اون ماده برای ما نیست و برای باکتری‌های روده است که اون ماده غذایی رو مصرف می‌کنند و خیلی هم خوبه. 

مشکل از ماست که می‌ریم که غذاها رو خراب می‌کنیم. مثلاً سیب هیچ مشکلی نداره. در صورتی که سیب هم فروکتوز داره. همین سیب رو اگر با دستگاه آب‌میوه‌گیری آبش رو بگیریم و فیبر (همون تفاله‌اش) رو دور بریزیم، اصلاً چیز خوبی نمیشه. فقط داستان اینه که فیبر رو نمی‌شه در فروشگاه نگه داشت و میگن فیبر غیرقابل حل یا Insoluble Fiber دیگه عمر قفسه‌ای (Shelf Life) نداره. در صورتی که آب سیب رو میشه به راحتی نگه‌داری کرد. و همین آب سیب هست که خیلی داستان داره. 

همین مسئله برای خیلی موادغذایی دیگه هم هست و اصلاً یکی از ویژگی‌های قند که باعث شده انقدر زیاد ازش استفاده بشه همین Shelf Lifeاش هست که به این سادگی‌ها خراب نمیشه (وگرنه سیب رو بذار تو فریزر ببین چطور مثل پودر میشه).

مثلاً برای انجیر که گفتی [اینجا](https://www.healthline.com/nutrition/figs-benefits) رو نگاه کن. انجیر کلی مواد معدنی و ویتامین و فیبر و چیزهای دیگه داره. خرما هم همین‌طوره. خلاصه هیچ کدوم از این‌ها [Empty Calorie](https://en.wikipedia.org/wiki/Empty_calories) نیستند و در کنار اون کالری‌ای که دارند، ریزمغذی‌های زیادی هم دارند و خواص زیادی هم براشون مطرح میشه. 

یا عسل هم همین‌طور. مثلاً این مقاله راجع به خواص عسل گفته: 

[Honey and Diabetes: The Importance of Natural Simple Sugars in Diet for Preventing and Treating Different Type of Diabetes - PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5817209/)

واقعیتش فکر نمی‌کنم این مواد غذایی چندان مشکل‌ساز باشند و مثل قند چیزی باشن که تو جون همه‌مون افتاده باشند، به چند دلیل:
- یکی این که معمولاً مصرف اکثر مردم برای این‌ها زیاد نیست و این‌طور نیست که پنهان باشه. یعنی شما اگر خرما یا عسل می‌خوری حواست هست که مثلاً ۲ تا خرما خوردی. 
- دوم این که همیشه و همه جا در دسترس نیستند. قند چیزیه که فراوان همیشه و همه‌جا هست ولی مثلاً انجیر فصل خودش رو داره. 
- سوم این که معلوم نیست چه درصدی از این مواد غذایی رو ما مصرف می‌کنیم و چه بخشی‌اش رو باکتری‌های روده. 

و این که چون داریم می‌بینیم که چقدر مصرف می‌کنیم حواس‌مون هست که مصرف‌مون خیلی هم زیاد نشه. ولی اگر مصرف‌شون بیش از حد بشه احتمالاً مشکل‌ساز بشن. مثلاً اگر کسی روزی ۵ تا خرما و ۲ تا قاشق عسل و ۱۰ تا انجیر هم بخوره و یه چایی نبات سفت هم روش بخوره، من چیز خوبی نمی‌تونم براش متصور بشم. بالاخره کبد هم توان پردازشی محدودی داره. تا یه حدی از فروکتوز (فکر می‌کنم ۱۲ گرم بود) رو می‌تونه پردازش کنه. اما بیشترش ممکنه داستان‌ساز بشه. 

به نظرم تو جاهای مختلف تغذیه‌ای کمی در مورد میوه زیاده‌روی شده؛ یعنی انقدر گفتند میوه خوبه که بعضی‌ها فکر می‌کنند اگر در کنار وعده‌های غذایی ۱۰ تا ۱۰ تا میوه بخورند، لابد خوبه. در صورتی که لزوماً این‌طور نیست. 

فعلاً در رابطه با مواد طبیعی مثل این‌ها به نتیجه قطعی نرسیدم (ببخشید که حرف زیاد زدم و چیز خاصی هم نگفتم). ولی چیزی که به نظرم قطعی هست نتایج تحقیقات راجع به قند افزودنی (یا همون ساکارز) هست که لینک‌هاش رو هم گذاشتم. 

به شخصه چون کتوژنیک گرفتم، فعلاً حذف‌شون کردم ببینم چی میشه ولی فکر می‌کنم در آینده به مقدار کم جاهای مختلف استفاده خواهم کرد.

**Reply by masoudnasrollahi** [reply]
ممنون به‌خاطر شیرینی کلامت. ایشالا که سلامت باشی
فقط اگه یه وقت خواستی حلوای من را بخوری، این حرف‌ها و نقل‌ها را بذار کنار؛ قشنگ با ولع انگشت بکش و چیزی از لذت کف دیس زندگی باقی نذار

**Comment by HashemAlavi** [comment]
سلام امیر عزیز

اول از همه ممنون بابت همه‌ی لینک‌ها و منابعی که معرفی کردی. من کلیّت قضیه رو تو بیوشیمی خونده بودم ولی هیچوقت اینطوری تحقیق نکرده بودم در موردش و صرفاً در مورد خودم سعی کردم رعایت کنم و قند نخورم. این سرنخ‌ها و لینک‌هایی که گذاشتی کمکم می‌کنه که بعداً بتونم راحت‌تر به مطالب دسترسی داشته باشم.

و اما نق‌های خودم:

مشکلی که من با قند دارم اینه که اکثراً نمی‌دونیم از کجا داریم می‌خوریم. من دور و برم زیاد می‌بینم کسایی که بخاطر مصرف زیاد قند (عمدتاً نوشابه و برنج) دارن خودشون رو نابود می‌کنن. مثلاً اخیراً یکی از بچه‌های فامیل‌مون که کلاس پنجمه کبد چرب گرفته. و همچنان هم نوشابه مصرف می‌کنه! غافل از اینکه کبد چرب مثل فشار خون که بهش اشاره کردی یجورایی قاتل خاموشه. یعنی ممکنه کبد از بین بره در طول زمان ولی متوجه نشی. و موقعی متوجه بشی که باید پیوند بشه.

اینا رو میشه تو سطح بزرگتری هم دید و اونم سواد اندک آدما در مورد موضوعات خیلی کاربردی پزشکی تو زندگی روزمره‌ست. مثلاً یک مورد دیگه‌اش که اگه بیشتر از قند آسیب‌زا نباشه کمتر نیست، مصرف بی‌رویه‌ی آنتی‌بیوتیکه که مقاومت باکتری‌ها رو می‌بره بالا (که البته بجز ایران کشورهای دیگه هم این مسئله رو دارن مثل چین یا بنگلادش). و جالبه که دکترها هم خودشون بخشی از این روند مخرب هستن. به هر حال بیزینسه دیگه!

ممنونم از مطلبی که نوشتی. منم گفتم حالا که بحثش شد چند خط از نق‌های خودم رو اینجا بنویسم.

**Reply by pourmand1376** [reply]
سلام هاشم جان. 

چه خوب اشاره‌ای کردی. اتفاقاً بحث باکتری‌های روده و عوامل تاثیرگذار روی اون‌ها مثل آنتی بیوتیک‌ها، Probiotics و Prebiotics هم اخیراً برام جذاب بوده و دارم سعی می‌کنم با مصرف سبزیجات متنوع و غذاهایی مثل ماست و غیره، ترکیب باکتری‌های روده‌ام رو تغییر بدم. راجع به تأثیراتی که روی سلامت چه روحی و روانی داره هم زیاد شنیدم. 

جالب برام اینه که خیلی از بیماری‌ها از رواج باکتری‌های بد تو روده ایجاد میشن و همین باکتری‌ها حتی می‌تونند تو جاهایی مثل افسردگی هم تأثیر داشته باشند. بعضی ها هم تعبیر جالبی دارند و ازشون به عنوان «مغز دوم» یاد می‌کنند. 

دیشب داشتم این ویدئو رو می‌دیدم که حتی می‌گفت جایگزینی مواد دفعی بدن آدم‌ها با هم چه تأثیرات شگرفی می‌تونه روی سلامتی داشته باشه. 

- [How the Gut Microbiome affects the Brain and Mind](https://www.youtube.com/watch?v=b4CBy0uVqRc)

خلاصه بدن هم سیستم پیچیده و به هم پیوسته‌ای هست و خیلی از مشکلات ما از بدغذایی شروع میشه. حالا جلوتر به تأثیر مخربی که قند روی همین باکتری‌های روده می‌گذاره هم می‌رسم و کلی دوباره تو سرش می‌زنم :))

**Comment by elhamghadiri** [comment]
خیلی مطلب جالب و مفیدی بود. دقیقا من هم اخیرا تو مطالعاتی که داشتم به این نکات اشاره کرده بودند که چربی سالم و غیراشباع اگه به میزان کافی مصرف بشه اتفاقا برای بدن خوبه و این قند (بخصوص قند مصنوعی) هست که مشکل زاست . من هم چند وقتی میشه که مصرف قند مصنوعی رو به نزدیک 0 رسوندم یعنی حتی سس هم مصرف نمیکنم، نون سفید و برنج هم شاید هفته‌ای یک تا دوبار و شاید هر از گاهی یک شیرینی یا بستنی (اگه به شدت هوس کنم) میخورم.

**Reply by pourmand1376** [reply]
خیلی عالیه! راجع به نون سفید و برنج سفید باهاتون موافقم. تو یزد هم به سختی میشه نون با آرد درست حسابی پیدا کرد و کلاً متأسفانه غذاهای کارخونه‌ای خیلی زیاد شده. 

اتفاقاً می‌خوام یه کم که بیشتر مطالعه کردم و مطمئن‌تر شدم مطلبی در «ستایش چربی‌ها» هم بنویسم. 

این‌طور که متوجه شدم مصرف چربی در آمریکا از ۱۹۵۰ به شدت کم‌ شده و به جای اون «هرم غذایی استاندارد» ترویج شده که خیلی از کارشناسان معقتدند بدترین روش غذا خوردن هست. نتایجش هم مشخصه و داریم می‌بینیم. 

اما راجع به چربی‌های اشباع که گفتید کلی داستان پشتش داره و این‌طور که من متوجه شدم هنوز به نتایج نرسیدند. یعنی یک آقایی به نام Keys اومد با تحقیقات Fake نشون داد که مصرف چربی‌های اشباع باعث افزایش مرگ و میر و بیماری‌های قلبی میشه و دیگه هر کی اومد بعداً بررسی کرد، همچین نتیجه‌ای گرفته نشد. 

این مقاله شارلاتان بازی‌هایی که در زمینه چربی اشباع صورت گرفته رو توضیح میده.

- [A short history of saturated fat: the making and unmaking of a scientific consensus - PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9794145/)

البته بحثی نیست که چربی اشباع LDL خون که به کلسترول بد معروفه رو افزایش میده. و خب کلسترول بد هم معروف هست که باعث بیماری‌های قلبی میشه. روی این دو تا گزاره هیچ بحثی نیست. اما مسئله اینه که تحقیقی تا حالا نتونسته نشون بده که مصرف چربی‌های اشباع باعث افزایش بیماری‌های قلبی میشه (یا حداقل من پیدا نکردم). در واقع مهم‌ترین بحثی که حول چربی‌های اشباع مطرح میشه همینه که اونم از Scientific Consensus برخوردار نیست. 

مشکل اینه که در رابطه با چربی‌های اشباع مسائل از جنس Correlation هست نه Causation. و جالب اینه که در تمام انواع چربی‌هایی که می‌خوریم مقداری چربی اشباع هست حتی در روغن زیتون و فقط درصد این هست که تغییر می‌کنه. یعنی یک سری غذاها درصد چربی‌ اشباع‌شون بیشتر هست و یک سری غذاها انواع دیگه‌ چربی‌شون که بر همین اساس دسته‌بندی‌شون می‌کنند. و خب اگر دقیق بشیم چربی هفت نوع داره که هر کدوم تأثیرات کاملاً متفاوتی دارند و تقسیم‌بندی چربی‌ها به سه نوع کمی ساده‌سازی هست. 

یک توضیحی که برای LDL مطرح میشه بحث این هست که LDL دو نوع داره که به sdLDL و lbLDL هست و sdLDL با بیماری‌های قلبی ارتباط داره که از قند میاد! و خب lbLDL از چربی اشباع میاد که برای قلب ضرری نداره. 

- [Saturated fats and cardiovascular health: Current evidence and controversies - PubMed](https://pubmed.ncbi.nlm.nih.gov/34649831/)

البته شما به هر پزشکی مراجعه کنید میگن که چربی اشباع بده و باعث گرفتگی رگ و عروق میشه ولی من شواهد محکمی پیدا نکردم. 

البته برای روغن‌های گیاهی و به طور خاص مصرف بیش از حد امگا ۶ شواهد زیادی هست. نسبت امگا ۳ به امگا ۶ خیلی مهمه و نباید مصرف امگا ۶ خیلی بیشتر بشه. اینه که به شخصه الان دارم تو مصرف روغن‌های گیاهی احتیاط می‌کنم. کما این که اون زمانی که چربی اشباع با رشوه شرکت‌های روغن‌های گیاهی بد اعلام شد، مصرف روغن‌های گیاهی به جای چربی اشباع توصیه شد. 

اینه که به شدت مشکوک هستم. حالا دنبال شواهدش هستم و هر وقت مطالب مستدل و مناسب‌تری پیدا کردم راجع بهش می‌نویسم. 

از نظر تاریخی هم برام جالبه. من از نظر تاریخی که بهش فکر می‌کنم اجداد ما خیلی چربی اشباع می‌خوردند یعنی حیوان رو می‌کشتند و همه اعضای بدنش رو می‌خوردند. خصوصاً به مغزش خیلی اهمیت می‌دادند و مدت زیادی از مغز و لاشه خواری تغذیه می‌کردند. نمی‌تونم از نظر تاریخی بفهمم که چرا بدن نتونسته طی صدها هزار سال تکامل به چربی اشباع عادت کنه! ولی تونسته طی ۱۰۰ سال به روغن‌های گیاهی (یا در کل Polyunsaturated fatها) عادت کنه. اینه که خیلی به کلیت قضیه مشکوکم.

**Comment by masoudnasrollahi** [comment]
<img width="1280" height="957" alt="خیانت کتاب سلامت و بهداشت اسید چرب ترانس" src="https://github.com/user-attachments/assets/bb7fd78a-4eb3-48c9-a52c-7a39b9baf8b5" />
سیستم آموزشی ما و کتاب‌های درسی! متأسفم...
